_posts/2022-12-20-trino-summit-2022-astronomer-recap.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: "Using Trino with Apache Airflow for (almost) all your data problems"
+author: "Philippe Gagnon, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/astronomer.jpg
+---
+
+As we close in on the final talks from [Trino Summit 2022]({% post_url 
+2022-11-21-trino-summit-2022-recap %}), this next talk dives into how to set up
+Trino for batch processing. Trino has historically been well-known for providing
+fast adhoc analytics queries as opposed to long-running, resource intensive
+queries commonly classified as batch or ETL queries. This is due to the fact
+that Trino kills queries that run out of resources in favor of lower latency.
+Earlier this year, Trino added
+[new features](https://trino.io/blog/2022/05/05/tardigrade-launch.html) that
+simplifies running batch queries on Trino by introducing a fault-tolerance mode
+that only requires restarting a task on failure rather than a stage or the
+entire query.
+
+Batch queries don't typically involve humans in the loop and run asynchronously.
+These tasks may depend on each other and have a complex workflow. To orchestrate
+this complexity, this talk covers using Airflow's new Trino integration to
+run Trino batch queries to solve (almost) all your data problems.
+
+<!--more-->
+
+{% youtube xKDN7RUJ5i4 %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Astronomer.pdf">
+  Check out the slides here!
+</a>
+
+## Recap
+
+This talk covers a fictional trading scenario that initially follows a
+traditional warehousing approach to storing their data. This architecture has
+data sources that are queried and submitted as raw data into a centralized
+warehouse. Within the warehouse itself, the raw data in transformed into data
+ready to be consumed.
+
+This model enforces centralization which requires one team that is removed from
+the business use case to run the platform and build the integration between
```

Comment:
I think the "one team that is removed from the business use case" is tough-ish to parse here. Maybe offset it with commas or reword.